components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/NodeIdModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.OpcUa.Publisher.Config.Models {
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Node id serialized as object
+    /// </summary>
+    [DataContract]
+    public class NodeIdModel {
+
+        /// <summary> Identifier </summary>
```

Comment:
>/// <summary> Identifier </summary> [](start = 8, length = 35)

nitpick -- you start a new summary style here

Reply:
Comments will be fixed in refactoring.